documentation/pom.xml

```diff
@@ -25,9 +25,9 @@
   </scm>
 
   <properties>
-    <asciidoctor.maven.plugin.version>2.2.2</asciidoctor.maven.plugin.version>
-    <asciidoctorj.pdf.version>1.6.2</asciidoctorj.pdf.version>
-    <asciidoctorj.version>2.5.3</asciidoctorj.version>
+    <asciidoctor.maven.plugin.version>2.2.4</asciidoctor.maven.plugin.version>
+    <asciidoctorj.pdf.version>2.3.9</asciidoctorj.pdf.version>
+    <asciidoctorj.version>2.5.10</asciidoctorj.version>
```

Comment:
It seems that the generated User-Guide is missing the Logos: The big one on the frontpage and the smaller ones on page headers and footers. Furtheron formatting (at least title page) has changed.

Reply:
Fixed with https://github.com/devonfw/solicitor/pull/219/commits/33f49d205be9b8f2d12017a3c042ed0c05780997